### DISCUSSION TITLE
Typo in File edit field

### DISCUSSION THREAD

**Discussion by 1reverseengineer** [community/community#19059] [Projects and Issues] [answered]
![image](https://user-images.githubusercontent.com/45468984/174856118-7548f776-223f-443c-93bd-e76283aabdec.png)
I opened another browser and observed the same issue. 
Perhaps more users can confirm on their side (Replication procedure: Open any repository file and click on the pen icon drop-down).
The capital (and additional) 'E' in the word 'file' is not intentional, I suppose.

**Comment by DiePfeiffe** [comment, marked as answer]
I think it's not a typo, but just missing space between "Edit this file" and "E" as a hotkey for this action. It is related to your browsers resolution/scaling. In my case, it's completely normal looking.
![image](https://user-images.githubusercontent.com/74821900/174859430-6bd9380d-9435-4c96-bea9-b3b962f86395.png)
